pages/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactElement } from "react";
+import Link from "next/link";
+import Layout from "@/components/common/Layout/Layout";
+import type { NextPageWithLayout } from "./_app";
+
+const HomePage: NextPageWithLayout = () => {
+  return (
+    <div>
+      Home 페이지
+      <ul>
+        <li>
+          <Link href="folder">폴더 페이지 이동</Link>
+        </li>
+        <li>
+          <Link href="shared">공유 페이지 이동</Link>
```

Comment:
요렇게 써주는게 더 직관적인 것 같구요!
```suggestion
          <Link href="/folder">폴더 페이지 이동</Link>
        </li>
        <li>
          <Link href="/shared">공유 페이지 이동</Link>
```

Reply:
이렇게 상수화해주면 좋습니다~
```suggestion
          <Link href={ROUTE_PATHS.FOLDER}>폴더 페이지 이동</Link>
        </li>
        <li>
          <Link href={ROUTE_PATHS.SHARED}>공유 페이지 이동</Link>
```